examples/scenario_1.py

```diff
@@ -87,7 +88,10 @@ def run_simulation(sim_start: str,
     computing_system = computing_system_sim.ComputingSystem(power_meters=power_meters)
 
     # Solar generator
-    data = pd.read_csv(solar_data_file, index_col="time", parse_dates=True)["solar"]
+    data : Union[pd.DataFrame, pd.Series] = pd.read_csv(
```

Comment:
interesting, what was the error here? I'd like to avoid boilerplate code to make some linter happy, so Id be curious what caused this

Reply:
Mypy raised the error that 'data' was of type 'Series' whereas 'pd.read_csv' returns a DataFrame. I couldn't find the location of the definition of data and I didn't want to create a bug in the code, therefore, Union seemed the best solution for that error.